instructure/canvas-ios
# Fix crash while generating preview in share extension

refs: MBL-16160
affects: Student
release note: Fixed a file share extension crash when submitting large files. Added file share extension previews for pdfs and screen recording videos.

test plan:
- Try to submit a large non-pdf file via share extension.
- Share extension should start without crashing.
- Try to submit a pdf and a screen recording file.
- Both should display previews in share extension dialog.
